docs/freqai-feature-engineering.md

```diff
@@ -161,6 +177,19 @@ You can ask for each of the defined features to be included also for informative
 In total, the number of features the user of the presented example strat has created is: length of `include_timeframes` * no. features in `feature_engineering_expand_*()` * length of `include_corr_pairlist` * no. `include_shifted_candles` * length of `indicator_periods_candles`
  $= 3 * 3 * 3 * 2 * 2 = 108$.
 
+
+ ### Gain finer control over `feature_engineering_*` functions with `metadata`
+
+ All `feature_engineering_*` and `set_freqai_targets()` functions are passed a `metadata` dictionary which contains information about the `pair`, `tf` (timeframe), and `period` that FreqAI is automating for feature building. As such, a user can use `metadata` inside `feature_engineering_*` functions as criteria for blocking/reserving features for certain timeframes, periods, pairs etc.
+
+ ```py
+def feature_engineering_expand_all(self, dataframe, period, **kwargs):
```

Comment:
I think it's necessary to change the method signature to includes the `metadata`param or change the example to read from `kwargs`, right?